server/src/api/read-extensions.ts

```diff
@@ -31,6 +31,10 @@ export function availableExtensions() {
       name: '@nativescript/schematics',
       description: 'NativeScript Support'
     },
+    {
+      name: '@ngrx/schematics',
+      description: 'NgRx state management generators'
+    },
     { name: '@ngrx/effects', description: 'NgRx side effect management' },
```

Comment:
I left the other extensions in here since they each have a single schematic that I'm guessing is poorly formed as they do not display in the list.

Reply:
Will do it when I get home. I saw after I submitted this that canAdd removes schematics named 'ng-add'